addons/mass_mailing/views/mailing_contact_views.xml

```diff
@@ -68,6 +68,10 @@
                     domain="[('opt_out', '=', False), ('is_blacklisted', '=', False), ('email_normalized', '!=', False)]"
                     invisible="not context.get('default_list_ids')"/>
                 <separator/>
+                <filter name="bounce" string="Bounce" domain="[('message_bounce', '>', 0)]"/>
```

Comment:
Name should be filter_bounce (and same for others), as by convention we add a filter_ prefix to xml id.